src/components/Label/Label.stories.js

```diff
@@ -2,52 +2,60 @@
 import React from 'react';
 import { storiesOf, action, linkTo } from '@kadira/storybook';
 import {
-   withKnobs,
-   text,
-   boolean,
-   number,
-   select,
+  withKnobs,
+  text,
+  boolean,
+  number,
+  select,
 } from '@kadira/storybook-addon-knobs';
 
 import Label from '.';
 
 const stories = storiesOf('Label', module);
 stories.addDecorator(withKnobs);
 stories.addDecorator(story => (
-   <div style={{ margin: '50px' }}>
-      {story()}
-   </div>
+  <div style={{ margin: '50px' }}>
+    {story()}
+  </div>
 ));
 
 const arrowOptions = {
-   left: 'Left',
-   right: 'Right',
-   top: 'Top',
-   bottom: 'Bottom',
+  left: 'Left',
+  right: 'Right',
+  top: 'Top',
+  bottom: 'Bottom',
 };
 
```

Comment:
Can we use `import { PositionEnum } from '../assets/constants'` for arrow options? The options basically represent all position properties.